### ISSUE TITLE
[Scheda 4] Esercizio 1

### ISSUE THREAD

**Issue by matypist** [sapienzastudentsnetwork/calcprob2223#23]
**Esercizio 1.** L’urna _A_ contiene 6 palline gialle e 4 palline rosse, mentre le urne _B_ e _C_ ne contengono 3 gialle e 7 rosse ciascuna. Viene scelta un’urna a caso fra _A, B, C_ e da questa ne vengono estratte casualmente ed in blocco tre palline.

Si ponga _E_ = {le tre palline estratte sono tutte rosse}

**Comment by i0nut02** [comment]
<img width="556" alt="Schermata 2022-10-25 alle 12 02 27" src="https://user-images.githubusercontent.com/99051485/197744871-72515cb0-19f9-4165-81d1-e74d8c27c153.png">

**Comment by matypist** [comment]
![image](https://user-images.githubusercontent.com/69389461/198892316-f539dc6f-90e5-4d41-8624-b180b14eb74b.png)

Trascrizione in LaTeX appunti di Bughi della correzione fatta a lezione dal professore:

![image](https://user-images.githubusercontent.com/69389461/215326753-9dd05df8-5e41-4b88-8bda-9c55464f9aa5.png)

**Comment by teoang** [comment]
> ![image](https://user-images.githubusercontent.com/69389461/198892316-f539dc6f-90e5-4d41-8624-b180b14eb74b.png)
> 
> ![image](https://user-images.githubusercontent.com/69389461/198892386-8c93f96a-4de1-4e1b-be87-076068262510.png)
> 
> _Trascrizione in LaTeX appunti di Bughi della correzione fatta a lezione dal professore_

Secondo me nel punto b) c'è una svista, io pensavo di calcolare il tutto facendo P(E)=P(E|A)*P(A)+P(E|B)*P(B)+P(E|C)*P(C) e in teoria mi viene lo stesso risultato di @i0nut02. Il primo calcolo è quello di @i0nut02, il secondo è quello mio e il terzo è quello di @matypist. Il binomial(10,3) mi sa che va a denominatore (nel primo addendo) e nel secondo addendo manca un binomial(7,3) a numeratore. 

<img width="1280" alt="Schermata 2022-10-30 alle 22 12 49" src="https://user-images.githubusercontent.com/107061030/198902007-18f6bf71-8820-4794-97c7-1ebf432169dd.png">

**Comment by matypist** [comment]
> > ![image](https://user-images.githubusercontent.com/69389461/198892316-f539dc6f-90e5-4d41-8624-b180b14eb74b.png)
> > ![image](https://user-images.githubusercontent.com/69389461/198892386-8c93f96a-4de1-4e1b-be87-076068262510.png)
> > _Trascrizione in LaTeX appunti di Bughi della correzione fatta a lezione dal professore_
> 
> Secondo me nel punto b) c'è una svista, io pensavo di calcolare il tutto facendo P(E)=P(E|A)*P(A)+P(E|B)*P(B)+P(E|C)*P(C) e in teoria mi viene lo stesso risultato di @i0nut02. Il primo calcolo è quello di @i0nut02, il secondo è quello mio e il terzo è quello di @matypist. Il binomial(10,3) mi sa che va a denominatore (nel primo addendo) e nel secondo addendo manca un binomial(7,3) a numeratore.
> 
> <img alt="Schermata 2022-10-30 alle 22 12 49" width="1280" src="https://user-images.githubusercontent.com/107061030/198902007-18f6bf71-8820-4794-97c7-1ebf432169dd.png">

Yeah, c'erano state una buona dose di errate interpretazioni della scrittura di Bughi in fase di trascrizione in LaTeX per quanto riguarda la soluzione alla lettera **b**, adesso ho corretto, grazie per la segnalazione!

**Comment by matypist** [comment]
I risultati che seguono dalle soluzioni di **i0nut02** [[🔗]](https://github.com/sapienzastudentsnetwork/calcprob2223/issues/23#issuecomment-1290299630) e dalle soluzioni degli appunti di Bughi della correzione fatta a lezione dal professore [[🔗]](https://github.com/sapienzastudentsnetwork/calcprob2223/issues/23#issuecomment-1296307958) sono equivalenti, possiamo pertanto assumere i risultati e le soluzioni come confermate e di conseguenza l'esercizio come risolto.
